docs/CaliptraIntegrationSpecification.md

```diff
@@ -624,7 +625,7 @@ The following table describes SoC integration requirements.
 | SRAMs                            | SoC shall write-protect fuses that characterize the SRAM.                                                                                                                                                                                 | Statement of conformance | Required for Caliptra threat model                |
 | SRAMs                            | SoC shall ensure SRAM content is only destroyed on powergood cycling.                                                                                                                                                                     | Statement of conformance | Functional (Warm Reset, Hitless Update)           |
 | SRAMs                            | SoC shall only perform SRAM repair on powergood events and prior to caliptra_rst_b deassertion.                                                                                                                                           | Statement of conformance | Functional (Warm Reset, Hitless Update)           |
-| Backend convergence              | Caliptra is validated and backend converged at 400MHz and at process nodes - TSMC 5nm, -- \<To be filled accurately\>                                                                                                                     |                          | Functional                                        |
+| Backend convergence              | Caliptra is validated and backend converged at 400MHz using an industry standard, moderately advanced technology node as of 2023 September                                                                                                |                          | Functional                                        |
```

Comment:
If this is only informational then it should be removed from "SoC integration requirements". If it's a requirement then it should be stated as (something like) "The Caliptra IP shall support frequencies between X MHz and Y MHz."